blog/docs/articles/configurable-runtimeclassnames.md

```diff
@@ -0,0 +1,62 @@
+# Configurable RuntimeClassNames
+
+**Author: Caleb Woodbine**
+
+Starting in Knative Serving v1.15, administrators are now able to configure default the RuntimeClassName field for deployments as default and via a Knative Service label selector.
+
+## Runtime Classes
+
+**What is a Runtime Class?**
+
+A Runtime Class for configuring the handler that runs a container, such as `runc`, `crun`, `runsc`, `nvidia` or `kata`.
+
+See documentation at the [Kubernetes docs here](https://kubernetes.io/docs/concepts/containers/runtime-class/).
+
+## Other configurations
+
+There are several feature flags in Knative Serving, one of which is enabling the field `.spec.template.spec.runtimeClassName` in Knative Service.
+
+This may be useful for self-service and is a helpful feature flag.
+
+See the documentation [here](https://knative.dev/docs/serving/configuration/feature-flags/#kubernetes-runtime-class).
+
+## Configuring Knative
+
+Knative is able to be configured with either the ConfigMaps if deployed with plain manifests or the KnativeServing resource if deployed with the operator. The following examples will be using just the plain manifests.
+
+See this example where Knative will configure deployments managed by Knative through Services to use Kata by default or gVisor when the Knative Service has a label matching `my-label=selector`:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-deployment
+  namespace: knative-serving
+data:
+  runtime-class-name: |
+    kata: {}
+    gvisor:
+      selector:
+        my-label: selector
+```
+
+The keys, like `kata` and `gvisor` must match existing Kubernetes RuntimeClasses.
+
+For Knative docs, [see here](https://knative.dev/docs/serving/configuration/deployment/#configuring-selectable-runtimeclassname).
+
+## Why is this important?
+
+RuntimeClasses enable several things, including:
+
+- security
+  - such as through Kata or gVisor 
+- functionality
+  - such as through GPU RuntimeClasses
+- 
```

Comment:
Nit: it seems like you left an extra point here
```suggestion
```

Reply:
Trying to remember what the other thing is. Good things come in threes!

Reply:
Also, looking at the preview render doesn't display nested points unfortunately

https://deploy-preview-6159--knative.netlify.app/blog/articles/configurable-runtimeclassnames/#why-is-this-important

Reply:
I vaguely recall running into this in the past, maybe using another space or two will fix the render?

Reply:
Ah huh! That did the trick. Cheers!